bread/bread.py

```diff
@@ -105,17 +102,19 @@ def dispatch(self, request, *args, **kwargs):
         return super(BreadViewMixin, self).dispatch(request, *args, **kwargs)
 
     def get_template_names(self):
-        # Is there a template_name_pattern?
+        # Return Django Vanilla templates (app-specific), then
+        #        Customized templates (via Bread object or Django settings), then
+        #        Django Bread templates
+        vanilla_templates = super(BreadViewMixin, self).get_template_names()
+        default_templates = ['bread/%s.html' % self.template_name_suffix]
         if self.bread.template_name_pattern:
-            return [self.bread.template_name_pattern.format(
+            custom_templates = [self.bread.template_name_pattern.format(
                 app_label=self.bread.model._meta.app_label,
                 model=self.bread.model._meta.object_name.lower(),
                 view=self.template_name_suffix
             )]
-        # First try the default names for Django Vanilla views, then
-        # add on 'bread/<viewname>.html' as a final possibility.
-        return (super(BreadViewMixin, self).get_template_names()
-                + ['bread/%s.html' % self.template_name_suffix])
+            return (vanilla_templates + custom_templates + default_templates)
+        return (vanilla_templates + default_templates)
```

Comment:
Can we remove these parentheses?  The `return` doesn't need them, and when I first scanned the code I thought we were constructing tuples until I looked closer.